src/orange/MatrixUtils.cc

```diff
@@ -16,6 +16,26 @@ using Mat3 = celeritas::SquareMatrixReal3;
 
 namespace celeritas
 {
+namespace
+{
+//---------------------------------------------------------------------------//
+//! Use fused multiply-add for floating point calculations
+template<class T, std::enable_if_t<std::is_floating_point<T>::value, bool> = true>
+T generic_fma(T a, T b, T y)
```

Comment:
I could probably rename this `fma`, add `CELER_FUNCTION` decorators, and put it in `Algorithms.hh`?

Reply:
It does make sense.  Then for consistent we will have to consider replacing the other use of `std::fma` (maybe even in non-templated context with `generic_fma`)

Reply:
Done in the follow-on PR.